packages/mwp-core/src/util/linkify.js

```diff
@@ -23,6 +23,16 @@ const getRelAttr = (followedExternalDomains, link, target) => {
 	}
 };
 
+export const getSubDomain = (url: string) => {
+	if (!url || !url.includes('//')) {
+		return 'www';
+	}
+
+	const [, domain] = url.split('//');
+	const [, , subdomain] = domain.split('.').reverse();
```

Comment:
Will it work if we just use first value from `domain.split('.')`? `const [subdomain] = domain.split('.');`